src/test/java/org/folio/inventory/domain/instances/DatesTest.java

```diff
@@ -7,11 +7,18 @@
 import org.junit.Test;
 import org.junit.runner.RunWith;
 
-import static org.folio.inventory.domain.instances.Dates.datesFromJson;
+import java.util.Map;
+import java.util.UUID;
+
+import static org.folio.inventory.domain.instances.Dates.convertToDates;
 import static org.folio.inventory.domain.instances.Dates.datesToJson;
+import static org.folio.inventory.domain.instances.Dates.retrieveDatesFromJson;
 import static org.hamcrest.MatcherAssert.assertThat;
+import static org.hamcrest.Matchers.in;
 import static org.hamcrest.Matchers.is;
 import static org.hamcrest.Matchers.nullValue;
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertNotNull;
```

Comment:
please double-check: some imports are not used.

Reply:
@Aliaksandr-Fedasiuk Done. Could you please double-check it?